### ISSUE TITLE
Austin Code COVID-19 Complaint Cases - Data Clean Up 

### ISSUE THREAD

**Issue by VictoriaODell** [open-austin/project-ideas#157]
### What problem are you trying to solve?
TLDR; Cleaning up COVID-19 311 complaint data to uncover and understand business compliance trends. We want to create a shared community understanding that can further guide action. 
<img width="1037" alt="Screen Shot 2020-07-21 at 4 02 41 PM" src="https://user-images.githubusercontent.com/10767430/88106780-a6e22880-cb6b-11ea-90a7-12a6f4ac1701.png">

The City of Austin has released [open data about COVID-19 complaints](https://data.austintexas.gov/Public-Safety/Austin-Code-COVID-19-Complaint-Cases/4p54-9544). However this set is missing some key data fields such as: 

- Type of business subcategory for the current "other" category
- Business Name
- Where the sources of the business name came from (either the address or the complaint notes or NA)

We are working to backfill about 5,000 rows of data AND request that 311 begin to regularly begin to document this data in their complaints about places of businesses moving forward. 

Our intention is to get a shared community understanding of what major businesses across the region are responding to guide further recommendations for policy and immediate actions. 

### Who will benefit (directly and indirectly) from your project?

The Community Resilience Trust(CRT), City, Community. 

CRT is a community group that has been meeting every weekday since the start of the COVID-19 outbreak in Austin. The group has organizations and individuals that span across the community with a shared goal of creating equity in Austin and starting with the current pandemic. 

So far the group has influenced the policy and position of various organizations and individuals and supported initiatives across the community. Projects from CRT aim to advocate for the best interests of the global majority in Travis County. 

### What other resources/tools are currently serving the same need? How does your project set itself apart?

The [city of Austin's database](https://data.austintexas.gov/Public-Safety/Austin-Code-COVID-19-Complaint-Cases/4p54-9544) is currently partially serving the need but does not include the desired rows. It's currently incomplete and there is some nuance on the release of new data collected and how it might affect small business. 

### Where can we find any research/data available/articles?

- [Current city of Austin's database](https://data.austintexas.gov/Public-Safety/Austin-Code-COVID-19-Complaint-Cases/4p54-9544)
- There is a presentation that can be shared with those interested in working on the project - Need to see if this is openly shareable
- We have already started the updated dataset but because of our intention to ensure we protect small businesses before releasing the data we are only sharing it with those that are helping hands. 
- [CRT Facebook](https://www.facebook.com/groups/190239735762438/)
- [CRT past email issues w/notes from daily calls](https://us4.campaign-archive.com/home/?u=f222b419313b8ed668a24a4a9&id=b0c6e33264)
- [Old CRT overview ](https://drive.google.com/open?id=1ec_YCb1WvZZG1KYF3hJuf_0u2cLd2EkD)

### What help do you need now?

We need some helping hands to get 5,000 rows of data updated. 

### What are the next steps (validation, research, coding, design)?

Research. We need people to help clean the sets up first so we can then move to analyzing and creating a presentation of the data with something like Tableau.

### How can we contact you outside of Github(list social media or places you're present)?

vodell@capitalcityinnovation.org 
victoria.d.odell@gmail.com
Open Austin Slack & Twitter - @VictoriaODell 

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [ ] Start the conversation about this idea on [Slack](https://openaustin-slackin.herokuapp.com/) Replace this link to the [#general](https://open-austin.slack.com/messages/C036FLME2) channel with your project's preferred channel.

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [x] Post an update at least once a month to this issue. Use [BASEDEF](https://github.com/open-austin/cheatsheet-basedef) for ideas, but it's ok even if your update is just "nothing new happened this month" or "we saw a small increase in traffic to our app this month". If there's no activity for two months, that's no problem, life happens. We'll just label this as `backlog` so others know you'll get back to it when you have the time. If nobody hears from you at all in more than two months, we may mark it as `abandoned` so that others can pick up this idea and run with it.
- [ ] Take 30 minutes to complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101) 
- [ ] Create a [GitHub repository](https://github.com/new). Ask for help setting up permissions if you want your repository to be within the [Open Austin Github organization](https://github.com/open-austin).
- [ ] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [ ] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.
- [ ] Create a [team](https://help.github.com/articles/creating-a-team/) for your core contributors
  - This will make it easier for you to manage your github repo access. People on a team have the same level of access. Admin access will allow your trusted contributors to make changes as needed. 
  - You can remove and add people to your team as needed. 
  - View some of our teams: [https://github.com/orgs/open-austin/teams](https://github.com/orgs/open-austin/teams)
  - Note: You can also allow collaborators outside of your team and give them more limited access.
- [ ]   Create a [user group](https://get.slack.help/hc/en-us/articles/212906697-User-Groups) in Slack so you can "@" your core contributors all at once, without bothering other people who use the Slack channel. You'll need permission from a Slack admin, so just mention @leadership on Slack to get this set up.
- [ ]   Create a [Google Drive](https://drive.google.com), [Dropbox](https://www.dropbox.com/), or other cloud storage to share larger files. Github and Data.World are good for code and data, respectively, especially when you need version control. But they're not good for very large files, documentation, articles, etc. A cloud storage option will allow you to easily share, create, and collaborate on documents with your team and help organize ideas and thoughts.
  - Doing this early on can help your team stay organized and to onboard new contributors who wouldn't have access to files you all have shared over email.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

- [ ] In your README, link to the [Open Austin Code of Conduct](https://www.open-austin.org/about/#code-of-conduct) or write your own code of conduct.
- [ ] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
- [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).
- [ ] Create an easily shareable project management artifact, like a [Civic Tech Canvas](https://cityofaustin.github.io/civic-tech-canvas/) or [Open Canvas](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/)
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [my project] to projects page`. An Open Austin leader will review this issue and post your project :balloon:
- [ ] [Tell the City of Austin](http://austintexas.gov/page/mobile-app-library). If your idea is in a shareable format and can benefit people around the city, go to that site and follow the instructions on the bottom of the page to showcase your work there.

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by VictoriaODell** [comment]
I'm submitting this on behalf of Janis Bookout and the Community Resilience Trust. I think I've captured everything we discussed but she can help flesh things out as needed. I just want to get the ball rolling 🧶

**Comment by VictoriaODell** [comment]
Just made a channel for this project! #project-crt-data on Slack

**Comment by VictoriaODell** [comment]
Project updates and chatter happening in #p-crt-data on Slack. @luqmaan has helped to add address data for businesses. 

2 Data parties have happened independently and two additional folks have expressed interest in helping via Open Austin. 

Here is a visualization shared in Slack done by a community member independently. 
https://public.tableau.com/profile/amy.r.wu#!/vizhome/Ironviz_15960326515960/COVID-19ComplaintsinAustin

**Comment by VictoriaODell** [comment]
Update! We got a little funding to help get all the data actually cleaned up since that was where we were having trouble finding volunteers. 

**Details below**

Seeking some folks for paid and volunteer work to join the CRT 3-1-1 COVID-19 Data Insights team! Our team training will be on November 11th from 6:00 pm - 7:30 pm.

:link: Here's a quick link to the form to apply:
https://forms.gle/vwugR2Nn2FJHxMWc9

:link: Here's a quick link to the blog post with additional details:
https://victoria-odell.medium.com/join-the-3-1-1-covid-19-data-insights-team-a2a2c7e03f01

Drop any questions you might have below
